Add sensors to the Traccar Server documentation (#31624)
home-assistant/home-assistant.github.io: source/_integrations/traccar_server.markdown

{% enddetails %}

### Sensor - Address

The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the address reported by the Traccar Server.

This entity is disabled by default.

{% configuration_basic %}
Name:

>   description: The name of the sensor will be set to what you have named it in Traccar Server followed by Address.

Reviewer: could you give an example of such a name?

Entity ID:

>   description: This will be a slugified version of the name.

Reviewer: I think an example could be helpful here, as well.

Author: As we can not guarantee this, I think adding that can cause more confusion.
If you have to sensors that want the same entity ID, the second one to be loaded will get a `_2` suffix.

Unique ID:
  description: This will be the unique ID of the device tracker in Traccar Server followed by `position_address`.
State:

>   description: This will be the address reported by the Traccar Server, if geo detection is not configured this will be unknown`.

Reviewer: ```suggestion
  description: This will be the address reported by the Traccar Server, if geofence is not configured, this will be unknown`.
```
Should this be geofence? This is the only occurrence of geo detection I find here. or is this something else?

Author: Geo detection is a setting that needs to be enabled on the server.

{% endconfiguration_basic %}

This entity does not have any attributes.

### Sensor - Altitude

The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the altitude reported by the Traccar Server.

This entity is disabled by default.

{% configuration_basic %}
Name:

>   description: The name of the sensor will be set to what you have named it in Traccar Server followed by Altitude.

Reviewer: could you give an example?

Entity ID:
  description: This will be a slugified version of the name.
Unique ID:
  description: This will be the unique ID of the device tracker in Traccar Server followed by `position_altitude`.
State:

>   description: This will be the altitude in meters (you can select a different unit in the entity options if you want).

Reviewer: ```suggestion
  description: This will be the altitude in meters. You can select a different unit in the entity options if you want.
```

{% endconfiguration_basic %}

This entity does not have any attributes.

### Sensor - Battery

> The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the battery percentage reported by the Traccar Server.

Reviewer: ```suggestion
The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the remaining battery percentage reported by the Traccar Server.
```

{% endconfiguration_basic %}

This entity does not have any attributes.

### Sensor - Geofence

The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the geofence reported by the Traccar Server.

This entity is disabled by default.

{% configuration_basic %}
Name:
  description: The name of the sensor will be set to what you have named it in Traccar Server followed by Geofence.
Entity ID:
  description: This will be a slugified version of the name.
Unique ID:
  description: This will be the unique ID of the device tracker in Traccar Server followed by `geofence_geofence`.
State:
  description: This will be geofence that the device is in, if you have overlapping geofences it will show the first one as reported by the Traccar Server.
{% endconfiguration_basic %}

This entity does not have any attributes.

### Sensor - Speed

The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the speed reported by the Traccar Server.

This entity is disabled by default.

{% configuration_basic %}
Name:

>   description: The name of the sensor will be set to what you have named it in Traccar Server followed by Speed.

Reviewer: An example could be helpful

Entity ID:
  description: This will be a slugified version of the name.
Unique ID:
  description: This will be the unique ID of the device tracker in Traccar Server followed by `position_speed`.
State:

>   description: This will be the speed of the device in knots (you can select a different unit in the entity options if you want).

Reviewer: ```suggestion
  description: This will be the speed of the device in knots. You can select a different unit in the entity options if you want.
```

{% endconfiguration_basic %}

This entity does not have any attributes.

### Sensor - Status

The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the status reported by the Traccar Server.

This entity is disabled by default.

{% configuration_basic %}
Name:
  description: The name of the sensor will be set to what you have named it in Traccar Server followed by Status.
Entity ID:
  description: This will be a slugified version of the name.
Unique ID:
  description: This will be the unique ID of the device tracker in Traccar Server followed by `devcie_status`.
State:

>   description: This will be one of the following; `offline`, `unknown`, `online`.

Reviewer: ```suggestion
  description: This will be one of the following: `offline`, `unknown`, `online`.
```

>   description: This will be one of the following: `offline`, `unknown`, `online`.

Reviewer: ```suggestion
  description: This will be one of the following; `offline`, `unknown`, `online`.
```
